tests/testthat/test-data_integrity.R

```diff
@@ -16,12 +16,14 @@ describe("data integrity", {
     }, no_check = TRUE)
   })
 
-  test_that('it crashes when trying to expand a table on new column when safe_columns is TRUE', {
-    dbtest::with_test_db({
-      lapply(dbListTables(test_con), function(t) dbRemoveTable(test_con, t))
-      cached_fcn <- cache(batch_data, key = c(key = "id"), c("model_version", "type"), con = test_con, prefix = prefix, safe_columns = TRUE)
-      cached_fcn(key = 5:1,  model_version, type)
-      expect_error(cached_fcn(key = 1:10, model_version, type, add_column = TRUE))
+  describe("when safe_column is TRUE", {
+    test_that('it crashes when trying to expand a table on new column', {
+      dbtest::with_test_db({
+        lapply(dbListTables(test_con), function(t) dbRemoveTable(test_con, t))
```

Comment:
@kirillseva Just did a little test. Tables persist between runs.  So this line is still needed.